lib/components/global.css

```diff
@@ -24,12 +24,14 @@
 }
 
 body {
+  color: #2a506f;
```

Comment:
It doesn't feel right that we are hardcoding the body and head colors, what is the reason for this?

Reply:
What's the alternative? I don't that the headers get the `text` color otherwise? Would using `injectGlobal` from styled-components be a better approach? That is by getting the theme properties.